src/features/MoviesSearch/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react'
+import styled from './styles.module.scss'
+
+function Main() {
+  // eslint-disable-next-line no-unused-vars
+  const [toggle, setToggle] = useState(false)
+
+  return (
+    <div className={`${styled.parent}`}>
```

Comment:
```suggestion
    <div className={styled.parent}>
```
it dies;t have to be a string

Reply:
> it dies;t have to be a string

i know but at that time i wanted to add another classes also. But fixed it

Reply:
updated